docs/processes/quality-assurance/templates/test-plan-template.md

```diff
@@ -0,0 +1,8 @@
+## **Test Plan Template**
```

Comment:
**nitpick:** Similar to suggestion above. Since this is a header, I think we don't need emphasis (ie. `**`). There is no visual difference 🤔